### PR TITLE
Added JSX Components For ProgressBar

### PR DESCRIPTION
**Pull request by B1TSH3LL**

### CODE REVIEW

**Comment by B1TSH3LL** [comment]
i haven't heard about tsx i was comfortable with jsx so i had developed the jsx component hope you understand please accept it and if you know about jsx please convert in your format

**Comment by vczb** [comment]
> i haven't heard about tsx i was comfortable with jsx so i had developed the jsx component hope you understand please accept it and if you know about jsx please convert in your format

Sorry man, your PR has a lot of issues. I can't accept it. Maybe you can find a project that you are more familiar

File: src/packages/Progressbar/package.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
+{
```
Comment (vczb) [inline src/packages/Progressbar/package.json:1]: why you create a package.json ?

File: src/packages/Progressbar/public/index.html [hunk 2]
```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
+    <meta name="theme-color" content="#000000" />
+    <link href="https://fonts.googleapis.com/css2?family=Nunito&display=swap" rel="stylesheet" />
+</head>
+
```
Comment (vczb) [inline src/packages/Progressbar/public/index.html:10]: ????

File: src/packages/Progressbar/src/styles.css [hunk 3]
```diff
@@ -0,0 +1,12 @@
+* {
+    font-family: "Nunito", sans-serif;
```
Comment (vczb) [inline src/packages/Progressbar/src/styles.css:2]: we are using styled components

File: src/packages/Progressbar/src/index.js [hunk 4]
```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+import ReactDOM from 'react-dom'
+
+import App from './App'
+const rootElement = document.getElementById('root')
+ReactDOM.render(
+  <React.StrictMode>
```
Comment (vczb) [inline src/packages/Progressbar/src/index.js:7]: ????

File: src/packages/Progressbar/src/app.js [hunk 5]
```diff
@@ -0,0 +1,24 @@
+import React, { useState, useEffect } from 'react'
```
Comment (vczb) [inline src/packages/Progressbar/src/app.js:1]: the dir name should be ProgressBar as well

File: src/packages/Progressbar/src/Progress.jsx [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+    width: PropTypes.string
+};
+
+ProgressBar.defaultProps = {
+    max: 100,
+    color: 'lightBlue',
+    width: '250px'
+};
+
+export default ProgressBar
```
Comment (vczb) [inline src/packages/Progressbar/src/Progress.jsx:56]: use typescript

File: src/packages/Progressbar/src/Progress.jsx [hunk 7]
```diff
@@ -0,0 +1,56 @@
...
+    value: PropTypes.number.isRequired,
+    max: PropTypes.number,
+    color: PropTypes.string,
+    width: PropTypes.string
+};
+
+ProgressBar.defaultProps = {
+    max: 100,
+    color: 'lightBlue',
+    width: '250px'
```
Comment (vczb) [inline src/packages/Progressbar/src/Progress.jsx:53]: use rem

File: src/packages/Progressbar/src/Progress.jsx [hunk 8]
```diff
@@ -0,0 +1,56 @@
...
+ProgressBar.propTypes = {
+    value: PropTypes.number.isRequired,
+    max: PropTypes.number,
+    color: PropTypes.string,
+    width: PropTypes.string
+};
+
+ProgressBar.defaultProps = {
+    max: 100,
+    color: 'lightBlue',
```
Comment (vczb) [inline src/packages/Progressbar/src/Progress.jsx:52]: use the theme that we already have

**Review by vczb — CHANGES_REQUESTED** [review]
out of the pattern, a lot of issues, to many commits

image

**Pull request closed without merging**